### ISSUE TITLE
Can I use this library in closed-source application and for commercial purpose?

### ISSUE THREAD

**Issue by sjahongir** [VahidN/PdfReport.Core#12]
I have not knowledge about licenses

**Comment by VahidN** [comment]
Yes. Here are the in use licences

Licenses
-----------------

| Library                 | License  |
| ----------------------- | :------: | 
| PdfReport.Core          | LGPLv2   |
|[iTextSharp.LGPLv2.Core](https://github.com/VahidN/iTextSharp.LGPLv2.Core)| LGPLv2 (It's not AGPL) |
|[EPPlus.Core](https://github.com/VahidN/EPPlus.Core)| LGPLv2|


And from the GNU website:

"The GNU Project has two principal licenses to use for libraries. One is the GNU Lesser GPL; the other is the ordinary GNU GPL. The choice of license makes a big difference: using the Lesser GPL **permits** use of the library in proprietary programs; using the ordinary GPL for a library makes it available only for free programs."
"The main difference between the GPL and the LGPL is that the latter can be linked to (in the case of a library, 'used by') a non-(L)GPLed program, and regardless of whether it is free software or proprietary software. This non-(L)GPLed program can then be distributed under any chosen terms if it is not a derivative work. If it is a derivative work, then the terms must allow "modification for the customer's own use and reverse engineering for debugging such modifications." Whether a work that uses an LGPL program is a derivative work or not is a legal issue. A standalone executable that dynamically links to a library is generally accepted as not being a derivative work. It would be considered a "work that uses the library" and paragraph 5 of the LGPL applies."

In simple terms, if you modify the source of an LGPL based library, you must distribute those modifications for the purpose of "reverse engineering" and "debugging".  That's it.
